dev/com.ibm.ws.security.fat.common.SSO.clientTests/fat/src/com/ibm/ws/security/SSO/clientTests/PrivateKeyJwt/PrivateKeyJwtClientTests.java

```diff
@@ -469,4 +484,127 @@ public void PrivateKeyJwtClientTests_noClientSSLRef() throws Exception {
 
     }
 
+    /**
+     * Test with the client config using the signature algorithm of RS256, and a private key jwt configured with RS256
+     * tokenEndpointAuthMethod="private_key_jwt"
+     * tokenEndpointAuthSigningAlgorithm="RS256"
+     * keyAliasName="altrs256"
+     * The client's alias name used with the keystore is different than the alias in the trust store
+     * The sslRef specified uses the key store file used to create the private, but the trust store uses a different alias - we
+     * should fail to get access to the app. The trustStoreRef will override the trust store from the sslRef, but they're both the
+     * same - so, it's hard to tell which is used, but for completeness, I'm testing it.
```

Comment:
Updated the comment indicating that the alias would be found in the truststore specified by the sslRef instead of what is specified by the truststoreref.